__tests__/app.test.js

```diff
@@ -149,7 +151,58 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad Request');
         })
     })
-})
-console.log('hello');
+    test('POST: 201 inserts a new comment to the db', () => {
+        const newComment = {
+            username: 'butter_bridge',
+            body: 'Awesome Article'
+          };
+          return request(app)
+            .post("/api/articles/1/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({body}) => {
+              expect(body.comment.comment_id).toBe(19);
+              expect(body.comment.author).toBe("butter_bridge");
+              expect(body.comment.body).toBe('Awesome Article');
+        });
+    })
+    test("POST: 400 sends status and error message when provided with wrong body (or no body)", () => {
+        const newComment = {
+            username: 'butter_bridge',
+          };
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(newComment)
+          .expect(400)
+          .then((response) => {
+            expect(response.body.msg).toBe("Bad Request");
+          });
+      });
+      test("POST: 400 sends a status and error message when given an invalid article_id", () => {
+        const newComment = {
+            username: 'butter_bridge',
+            body: 'Awesome Article'
+          };
+        return request(app)
+          .post('/api/articles/not_an_id/comments')
+          .send(newComment)
+          .expect(400)
+          .then((response) => {
+            expect(response.body.msg).toBe("Bad Request");
+          });
+      });
+      test("POST: 500 sends a status and error message when given a valid but non existing article id", () => {
```

Comment:
This shouldn't be a 500, it should be a 404. Whenever something is valid but non-existent, it means it could be valid in the future, so we should send a 404 👍